ui/components/MesheryAdapterPlayComponent.js

```diff
@@ -1068,7 +1069,7 @@ class MesheryAdapterPlayComponent extends React.Component {
                         {adapterChip}
                       </div>
                       <div className={classes.inputContainer}>
-                        <ReactSelectWrapper
+                        <ReactSelectWrapper style={{ width : "32.5%" }}
```

Comment:
I don't see style provided as a prop being used by this component: https://github.com/meshery/meshery/blob/0d8bc606ca5af6d65327b64b1831aac4038b1180/ui/components/ReactSelectWrapper.js
Probably, this change won't have any effect.

Reply:
Only These props are used by the `ReactSelectWrapper`

https://github.com/meshery/meshery/blob/0d8bc606ca5af6d65327b64b1831aac4038b1180/ui/components/ReactSelectWrapper.js#L133

You have to make changes in the component to access the prop provided by you. wrt to where you want to pass that prop in which react element.

Reply:
@Akshit42-hue was this comment addressed?

Reply:
If so, resolve comment and confirm fix, please.

Reply:
Yes, I have fix it